hack/machineset.sh

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
+
+# machineset.sh - apply/delete the Windows MachineSet for Windows Machine Config Operator
+#
+# USAGE
+#    machineset.sh apply/delete
+# OPTIONS
+#    $1      Action       apply/delete the MachineSet
+# PREREQUISITES
+#    oc                   to apply MachineSets on the cluster(cluster should be logged in)
+#    aws                  to fetch Windows AMI id for AWS platform (only required for clusters running on AWS)
+
+ACTION=$1
+if [[ ! "$ACTION" =~ ^apply|delete$ ]]; then
+    error-exit "Action (1st parameter) must be \"apply\" or \"delete\""
+fi
+
+# Creates a MachineSet for AWS Cloud Provider
+get_aws_ms(){
+
+local infraID=$1
+local region=$2
+local az=$3
+
+# get the AMI id for the Windows VM
+ami_date="2020.06.10"
```

Comment:
Is this Oct 10th or June 10th? :-) We do need to pick the AMI which has the VXLAN patch present.

Reply:
My Bad, i thought this one is the latest. 
We have June 10, Sept 9, Aug 12, July 15 as our options. 
I'll replace this with the latest one i.e. Sept 9

Reply:
This is based on the fact that i am considering the middle part is month and not date. I'm confused as well....

Reply:
OK, once the confusion about the date format is resolved, please ensure you pick one on or after Aug 12, 2020. Anything before that does not have VXLAN patch. Latest is good too.